desktop-app/src/store/index.ts

```diff
@@ -138,6 +138,27 @@ const schema = {
         type: 'boolean',
         default: false,
       },
+      guides: {
+        type: 'array',
+        items: {
+          type: 'object',
+          properties: {
+            positions: {
+              type: 'array',
+              items: {
+                type: 'number',
+              },
+            },
+            is_vertical: {
+              type: 'boolean',
+            },
+            resolution: {
+              type: 'string',
+            },
+          },
+          default: {},
+        },
+      },
```

Comment:
Can you add a default for the `userPreferences.guides` field as a `[]` too? 

As on the first run, I'm getting this error:
<img width="952" alt="Screenshot 2023-11-25 at 7 34 02 PM" src="https://github.com/responsively-org/responsively-app/assets/1283424/1349d845-4d92-4982-bc82-e5d567cfe9fe">

Reply:
ok, done.